Source/WebCore/PAL/pal/spi/mac/NSMenuSPI.h

```diff
@@ -79,6 +79,16 @@ enum {
 - (BOOL)_containsItemMatchingEvent:(NSEvent *)event includingDisabledItems:(BOOL)includingDisabledItems;
 @end
 
+#if ENABLE(CONTEXT_MENU_IMAGES_FOR_INTERNAL_CLIENTS)
+@interface NSMenuItem (Staging_139537720)
```

Comment:
Nit - rdar://139537720 seems to be the radar for this WebKit change. I think you meant rdar://138651669?

```suggestion
@interface NSMenuItem (Staging_138651669)
```